### PR TITLE
Fix and expand "unpack" usage docs.

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

**Comment by danielballan** [comment]
Thanks for the review. I learned since that Guillaume still hasn’t gotten this to work for him, so there may be fixes or clarifications to come.

File: docs/source/usage.rst [hunk 1]
```diff
@@ -93,43 +93,54 @@ characters) entered at the command prompt...
...
+
+.. code:: bash
+
+   databroker-unpack path/to/directory_from_pack my_data
+
+Then the newly "unpacked" catalog (e.g. ``my_data``) will show in
+
+.. code:: bash
+
+   databroker --list-catalogs
```
Comment (mrakitin) [inline docs/source/usage.rst:113]: Is it `databroker` or `databroker-pack`?

File: docs/source/usage.rst [hunk 2]
```diff
@@ -93,31 +93,54 @@ characters) entered at the command prompt...
...
-and
+and can be accessed like
+
+.. code:: python
+
+   >>> import databroker
+   >>> db = databroker.catalog['my_data'].get()
+
+This catalog, ``db``, contains the packed Runs, which can be accessed in the
+usual way like ``db[<uid>]``, ``db[<scan_id>]``, ``db[-1]``, or fully
```
Comment (mrakitin) [inline docs/source/usage.rst:123]:
```suggestion
usual way like ``db['<uid>']``, ``db[<scan_id>]``, ``db[-1]``, or fully
```

File: docs/source/usage.rst (hunk 1 above)
Comment (mrakitin) [reply docs/source/usage.rst]: For reference: https://github.com/bluesky/databroker-pack/blob/f970c72db82370f7216e14cf597434b548c3a6c6/databroker_pack/commandline/pack.py#L96

Comment (mrakitin) [reply docs/source/usage.rst]: @danielballan, thanks for fixing it via 48417b3.

**Pull request merged**